packages/jsii/lib/assembler.ts

```diff
@@ -277,12 +278,16 @@ export class Assembler implements Emitter {
     const validator = new Validator(this.projectInfo, assembly);
     const validationResult = validator.emit();
     if (!validationResult.emitSkipped) {
-      const assemblyPath = path.join(this.projectInfo.projectRoot, '.jsii');
-      LOG.trace(`Emitting assembly: ${chalk.blue(assemblyPath)}`);
-      fs.writeJsonSync(assemblyPath, _fingerprint(assembly), {
-        encoding: 'utf8',
-        spaces: 2,
-      });
+      LOG.trace(
+        `Emitting assembly: ${chalk.blue(
+          path.join(this.projectInfo.projectRoot, SPEC_FILE_NAME),
+        )}`,
+      );
+      writeAssembly(
```

Comment:
I put this in to "prove" that the new module was exporting functions correctly, but if you'd rather this be a part of the future PR that will update the rest of the libraries to use this module, then I can go ahead and revert the changes to `jsii`.

Reply:
Nah this is cool - this way you're not introducing (temporary) deadweight.